lnbits/core/static/js/wallet.js

```diff
@@ -723,27 +723,34 @@ new Vue({
     updateWallet: function (data) {
       LNbits.api
         .request('PATCH', '/api/v1/wallet', this.g.wallet.adminkey, data)
-        .then(res => {
+        .then(_ => {
           this.$q.notify({
             message: `Wallet updated.`,
             type: 'positive',
             timeout: 3500
           })
-          LNbits.href.updateWallet(
```

Comment:
- this used to work as a redirect `window.location.href`. Is it no longer needed?

Reply:
it is basically just a reload to change the name, i think its unecessary

Reply:
fixed